src/ape_ethereum/private_mempool.py

```diff
@@ -0,0 +1,26 @@
+from typing import Optional
+
+from ape.types.private_mempool import Bundle, SimulationReport
+from ape.utils.basemodel import ManagerAccessMixin
+
+
+class PrivateMempoolAPI(ManagerAccessMixin):
+    """
+    A wrapper around MEV APIs available through flashbots or reth.
+    """
+
+    def simulate_bundle(
```

Comment:
We don't necessarily have to match 1:1 the naming of the lower-level RPC

Considering we have also have `send_private_transaction` which some providers take advantage of (most notably Alchemy, but in theory I think any node thaf supports the `mev_` namespace exposes this?), how might that work here with `simulate_bundle`?

Should we migrate `send_private_transaction` here? Or add this function to `ProviderAPI` using default of `NotImplemented`?

What about adding `send_bundle` too?

Also, what do you think about `simulate_transaction_bundle`/`send_transaction_bundle`?